### PR TITLE
Changed the header style

### PR DESCRIPTION
**Pull request by Markol17**
I changed the styling of the header. It's still not done tho.

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -25,12 +25,19 @@ const Header = ({ siteTitle }) => {
...
     <Navbar fixedToTop>
       <NavbarGroup align={Alignment.LEFT}>
-        <NavbarHeading>
-          <Link to="/" style={{ textDecoration: 'none' }}>
-            <Button className={Classes.MINIMAL} text={siteTitle} />
-          </Link>
-        </NavbarHeading>
-        <NavbarDivider />
+
+        <div className={"container"}>
```
[the remark this answers is not in the document]
Comment (Markol17) [reply src/components/header.js]: ok will do

File: src/components/header.js [hunk 2]
```diff
@@ -43,8 +50,10 @@ const Header = ({ siteTitle }) => {
       </NavbarGroup>
     </Navbar>
   );
+
```
[the remark this answers is not in the document]
Comment (Markol17) [reply src/components/header.js]: ok I use atom but pretty sure I can install a prettier plugin that solves that.

[the remark this answers is not in the document]
Comment (ianDesson) [reply src/components/header.js]: you can also get the prettier cli tool from npm https://prettier.io/docs/en/cli.html

[the remark this answers is not in the document]
Comment (Markol17) [reply src/components/header.js]: ok thanks

[the remark this answers is not in the document]
Comment (Markol17) [reply src/components/header.js]: I just checked there is an atom package  for prettier I will install it

**Pull request closed without merging**